base_layer/wallet/src/wallet.rs

```diff
@@ -445,6 +445,7 @@ where
             script_lock_height,
             covenant,
             encrypted_value,
+            MicroTari::zero(),
```

Comment:
Dont default here, its going to break, import it from the utxo. This info comes from a utxo

Reply:
It should have